libarchive/archive_read_disk_windows.c

```diff
@@ -2438,6 +2438,7 @@ archive_read_disk_entry_from_file(struct archive *_a,
 		return (ARCHIVE_OK);
 	}
 
+	r = ARCHIVE_OK;
```

Comment:
Note that this is consistent with what `archive_read_disk_entry_from_file` does in `archive_read_disk_entry_from_file.c`: https://github.com/libarchive/libarchive/blob/9ba8292aafacb7de8ff3629e823a862aa2ec569f/libarchive/archive_read_disk_entry_from_file.c#L292

Before this point in time, `r` is used as a `BOOL`, which doesn't make sense to leave uninitialized anyway